src/python/supply/supply.go

```diff
@@ -284,8 +284,16 @@ func (s *Supplier) InstallPython() error {
 	}
 
 	version := regexp.MustCompile(`\d+\.\d+`).FindString(dep.Version)
-	if err := os.Setenv("CFLAGS", fmt.Sprintf("-I%s", filepath.Join(s.Stager.DepDir(), "python", "include", fmt.Sprintf("python%s", version)))); err != nil {
-		return err
+
+	//Remove once Python 3.7 is out of support (June 2023)
+	if version == "3.7" {
```

Comment:
I think it might be slightly cleaner to extract just the directory naming logic into the conditional e.g.:

```

dir := fmt.Sprintf("python%s", version)

if version == "3.7" {
  dir = fmt.Sprintf("python%sm", version)
}

if err := os.Setenv("CFLAGS", fmt.Sprintf("-I%s", filepath.Join(s.Stager.DepDir(), "python", "include", dir))); err != nil {
  return err
}
```

But I think for just a couple of months we can leave it as-is.

Reply:
Happy to still make this change in a separate PR if you think it's worth it @robdimsdale